src/clincoded/static/components/variant_central/interpretation/segregation/evidenceSheet.js

```diff
@@ -16,6 +16,14 @@ import { external_url_map } from '../../../globals';
 import { RestMixin } from '../../../rest';
 import * as curator from '../../../curator';
 
+// Define an HPO class to save HPO data and pass to tables
+class Hpo {
```

Comment:
No particular reason besides that it seemed more concise than Object.create().. etc. But since the hpo object has only two properties, just went with `const hpo = {......}` inside of `lookupTerm()`